ProcessQueue/__init__.py

```diff
@@ -508,21 +525,23 @@ def reuse_existing_value_concepts(new_values_map,content_type,api_url,headers):
     # active reports is list of report ids that are not archived
 
     # map value id to active scan report
-    #table_id_to_scanreport_map ={str(element.get("id", None)): str(element.get("scan_report", None)) for element in tables}
     table_id_to_active_scanreport_map = {str(element["id"]): str(element["scan_report"]) for element in tables if str(element["scan_report"]) in active_reports}
     field_id_to_active_scanreport_map ={str(element["id"]): table_id_to_active_scanreport_map[str(element["scan_report_table"])] for element in fields if str(element["scan_report_table"]) in table_id_to_active_scanreport_map}
-    #field_id_to_active_scanreport_map ={str(element.get("id", None)): table_id_to_scanreport_map[str(element.get("scan_report_table", None))] for element in fields if  str(table_id_to_scanreport_map[str(element.get("scan_report_table", None))]) in active_reports}
+
     value_id_to_active_scanreport_map = {str(element["id"]): field_id_to_active_scanreport_map[str(element["scan_report_field"])] for element in value_names if str(element["scan_report_field"]) in field_id_to_active_scanreport_map}
     value_names = [item for item in value_names if str(item["id"]) in value_id_to_active_scanreport_map]
     print("VALUES TO SCAN REPORT",value_id_to_active_scanreport_map)
     print("FILTERED VALUES",value_names)
 
     # 
-    existing_mappings=[(value['value'],value_id_to_concept_map[str(value['id'])],value['id']) for value in value_names]
+    existing_mappings=[(value['value'],value_id_to_concept_map[str(value['id'])],value['id'],value["value_description"],field_id_to_name_map[str(value["scan_report_field"])]) for value in value_names]
```

Comment:
I think it would be clearer if `existing_mappings` is a `list(dict)` rather than `list(tuple)` so that you can then say `mapping["value_description"]` below rather than `mapping[3]` etc. The same is also true of the equivalent variable in the field function.